app/lib/frontend/templates/tabs.dart

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:convert';
+
+import 'package:meta/meta.dart';
+
+import '_cache.dart';
+
+/// Renders the `views/pkg/tabs.mustache` template.
+String renderPkgTabs(List<Tab> tabs) {
```

Comment:
Maybe rename it `renderTabs`, or even:`Tab.render(List<Tab> tabs)` as a static method.